Conjugar/UIView+Accessibility.swift

```diff
@@ -0,0 +1,15 @@
+import UIKit
+
+extension UIView {
+    func setAccessibilityLabelInSpanish(_ label: String, region: String = Current.settings.region.accent) {
```

Comment:
These functions should go in `UIViewExtensions.swift`. As an aside, I see that the name of the file at the top of that file needs an `s`.

I recognize that putting specific extensions in files with names like `UIView+Accessibility.swift` is a widespread practice, but I prefer to gather all extensions for particular types in files with names like `UIViewExtensions.swift` and `URLSessionExtension.swift`.

Reply:
Okay, I added them there, but with a mark to distinguish the Accessibility section of the file. I enjoy that approach, since it joins the both worlds.

Reply:
I am not opposed to `MARK` comments in the abstract, but I don't use them elsewhere in the app, so I will remove them in a separate commit.